io.github.bkbilly.lnxlink.metainfo.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+  <content_rating type="oars-1.0" />
+  <id>io.github.bkbilly.lnxlink</id>
+  <name>LNXlink</name>  
+
+  <metadata_license>MIT</metadata_license>
+  <project_license>MIT</project_license>
+  <developer id="io.github.bkbilly">
+    <name>bkbilly</name>
+  </developer>
+  <url type="homepage">https://bkbilly.gitbook.io/lnxlink</url>
+
+  <summary>Manage Linux machine with MQTT</summary>
+  <description>
+    <p>
+      LNXlink is a Linux companion app that seamlessly integrates your system with external applications like Home Assistant. It uses MQTT, a lightweight messaging protocol, to enable real-time data exchange and remote control capabilities.
+    </p>
+    <p>Features:</p>
+    <ul>
+      <li>Sensors: Automatically discover sensors that monitor and control the system.</li>
+      <li>Home Assistant: Uses MQTT Autodiscovery to create entities and shows if update is required.</li>
+      <li>No sudo required: No need to be root user to install and use, unless used on server setup.</li>
+      <li>Easily expanded: Any new module is automatically imported and custom modules can be added.</li>
+    </ul>
+  </description>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/bkbilly/lnxlink/master/images/preview.png</image>
```

Comment:
screenshot should be from a tag or a commit not `master`

Reply:
I've added the latest tag.